cypress/integration/HelpWithTechnologyPage.test.js

```diff
@@ -0,0 +1,34 @@
+describe("Help with technology page", () => {
+  beforeEach(() => {
+    cy.visit("/teachers/help-with-technology/");
+    cy.shouldHaveHeaderAndFooter();
+    cy.shouldHaveBreadcrumbs();
+  });
+
+  it("Shows the Help with technology page", () => {
+    cy.shouldHaveHeading(
+      "Get help with technology for remote education during coronavirus (COVID-19)"
+    );
+    cy.contains("Previous").should("exist");
+    cy.contains(
+      "Safeguarding and remote education during coronavirus (COVID-19)"
+    ).should("exist");
+  });
```

Comment:
The reason for this test?

Is it to test the content pagination?

If so, perhaps we should split this into a helper test to check that Next goes to an expected page, and previous go to an expected page. 

```suggestion
  });
  
  it("Has pagination", () => {
    cy.shouldHavePagination(
      "/teachers/safeguarding-and-remote-teaching-during-coronavirus-covid-19/",
      "Safeguarding and remote education during coronavirus (COVID-19)",
      "Previous"
    );
  });
```

in the commands file

```javascript
Cypress.Commands.add("shouldHavePagination", (href, text, direction) => {
  cy.get(".gem-c-pagination").within(() => {
    cy.get(`a.gem-c-pagination__link[href="${href}"]`).should($el => {
      expect($el).to.contain.text(text);
      if(direction) {
        expect($el).to.contain.text(direction);
      }
    });
  });
});
```

Does this make sense to have a test target this better?

Reply:
Agree, It looks more better.

Reply:
Agree, It looks more better.